pkg/printers/internalversion/describe_test.go

```diff
@@ -2248,6 +2249,7 @@ func TestDescribePodSecurityPolicy(t *testing.T) {
 			Name: "mypsp",
 		},
 		Spec: policy.PodSecurityPolicySpec{
+			AllowedUnsafeSysctls: []string{"kernel.*", "net.ipv4.ip_local_port_range"},
```

Comment:
Can you add test for `ForbiddenSysctls` field as well? Just in case it gets removed by accident?

Reply:
Will do.